spanner/ddl/ast.go

```diff
@@ -112,7 +174,12 @@ func (c Config) quote(s string) string {
 // comment. These are returned as separate strings to support formatting
 // needs of PrintCreateTable.
 func (cd ColumnDef) PrintColumnDef(c Config) (string, string) {
-	s := fmt.Sprintf("%s %s", c.quote(cd.Name), cd.T.PrintColumnDefType())
+	var s string
+	if c.TargetDb == "experimental_postgres" {
```

Comment:
Use TARGET_EXPERIMENTAL_POSTGRES constant?

Reply:
We couldn't use the constant before because of circular dependency. Have now moved all constants to a top-level common package and now have updated to use constant as well.